### PR TITLE
Closure app upgrade to new UI

### PR DESCRIPTION
**Pull request by chrisprice**
Based on #1110

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks :)

You can't ignore the whole `node_modules` folder, see the other UI upgrade PRs. You also need to include the required files for running.

**Comment by chrisprice** [comment]
Good point! All done :)

Out of curiosity, how come the files are explicitly not ignored rather than force added?

File: examples/closure/.gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+node_modules/.bin
+
+node_modules/plovr
+
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/*
+!node_modules/todomvc-common/base.css
+!node_modules/todomvc-common/base.js
```
Comment (arthurvr) [inline examples/closure/.gitignore]: No big deal but maybe give this a final newline, as stated by our editorconfig: https://github.com/tastejs/todomvc/blob/master/.editorconfig#L8

Comment (passy) [inline examples/closure/.gitignore]: :+1:

**Comment by chrisprice** [comment]
Fixed.

**Comment by sindresorhus** [comment]
> Out of curiosity, how come the files are explicitly not ignored rather than force added?

No special reason. Could have done either. This one is just more explicit.

**Pull request merged**